spec/Vehicle/Vehicle.vspec

```diff
@@ -184,16 +184,53 @@ Speed:
   description: Vehicle speed.
 
 TravelledDistance:
+  deprecation: V3.1 moved to Vehicle.TraveledDistance
   datatype: float
   type: sensor
   unit: km
-  description: Odometer reading, total distance travelled during the lifetime of the vehicle.
+  description: Odometer reading, total distance traveled during the lifetime of the vehicle.
 
-TripMeterReading:
+TraveledDistance:
+  datatype: float
+  type: sensor
+  unit: km
+  description: Odometer reading, total distance traveled during the lifetime of the vehicle.
+
+TraveledDistanceSinceStart:
+  datatype: float
+  type: sensor
+  unit: km
+  description: Distance traveled since start of current trip.
+  comment: A new trip is considered to start when engine gets enabled (e.g. LowVoltageSystemState in ON or START mode).
+           A trip is considered to end when engine is no longer enabled.
+           The signal may however keep the value of the last trip until a new trip is started.
+
+StartTime:
+  datatype: string
+  type: attribute
+  default: '0000-01-01T00:00Z'
```

Comment:
At last meeting we discussed a bit on what an attribute is, and if this signal makes sense to be an attribute as it is only set once per ignition cycle. I changed the PR to use attribute as type, but then I found that we say in documentation that an attribute must have a default value. Although not explicitly said I assume it originates from an idea that an attribute always will have a value, i.e. if you do a Read you will always get a value (if the signal is defined), compared to sensors where you might get an answer that no value is available.

We can discuss if we want to keep this restriction for attributes - for now selecting a year 0 as default value, if used indicating that vehicle has never been started or that latest start time is unknown